pkg/storage/storage.go

```diff
@@ -298,3 +298,18 @@ func Mount(client *occlient.Client, path string, storageName string, componentNa
 	}
 	return nil
 }
+
+// Gets the storageName mounted to the given path in the given component and application
+// GetStorageNameFromMountPath returns the name of the storage or the error
+func GetStorageNameFromMountPath(client *occlient.Client, path string, componentName string, applicationName string) (string, error) {
+	storages, err := List(client, componentName, applicationName)
+	if err != nil {
+		return "", errors.Wrap(err, "unable to list storages")
```

Comment:
storage not storages

Reply:
@cdrage But we refer to one PVC as storage

Reply:
@mik-dass In English, we don't use the plural "storages". It's just "unable to list storage" https://en.wiktionary.org/wiki/storage

Reply:
@cdrage I know but a single PVC is called storage and all of them together is also called storage. It seemed confusing. So I added the 's' to distinguish. Anyways updating it.